hangman.py

```diff
@@ -0,0 +1,250 @@
+# Problem Set 2, hangman.py
+# Name: Zlotenko Bohdana
+# Students group: KM-04
+# Collaborators: Alexey Garmash
+# Time spent: really a lot
+
+import random
+from string import ascii_lowercase
+
+WORDLIST_FILENAME = "words.txt"
+VOWELS = ('a', 'e', 'i', 'o', 'u')
+UNKNOWN_LETTER = "_ "
+HINT = "*"
+
+warnings = 3
+guesses = 6
+letters = ascii_lowercase
+letters_guessed = []
+word_letters_guessed = []
```

Comment:
There's no need in having 2 different collections of letters, you can implement your code using only one collection